### ISSUE TITLE
`stringAttr` cuts values

### ISSUE THREAD

**Issue by miniBill** [ymtszw/elm-xml-decode#8]
Minimal example (runnable in repl):

```Elm
import Xml.Decode as XD
XD.run (XD.stringAttr "url") "<root url=\"a&amp;b\"></root>"
```

Expected: `"a&b"`
Actual: `"&b"`

**Comment by ymtszw** [comment]
Will look at it

**Comment by miniBill** [comment]
I realized it's not a problem with `elm-xml-decode` itself: https://github.com/jinjor/elm-xml-parser/issues/13

**Comment by ymtszw** [comment]
@miniBill I see. Thanks for inspecting

**Comment by ymtszw** [comment]
Since jinjor is inactive for a while, I'm considering vendoring jinjor/elm-xml-parser inside this package with open pull-requests applied to fix this issue. (Like [dillonkearns/elm-markdown did](https://github.com/dillonkearns/elm-markdown/blob/7.0.0/src/HtmlParser.elm).)
Will work on it if I got time, but anyone else picking up the work it would be greatly appreciated.

**Comment by miniBill** [comment]
I've fixed the issue in my fork, created a PR in #15

**Comment by ymtszw** [comment]
Much appreciated! Will create release later

**Comment by ymtszw** [comment]
Released in [3.2.2](https://package.elm-lang.org/packages/ymtszw/elm-xml-decode/3.2.2/)
